src/utils/__tests__/changelog.test.ts

```diff
@@ -411,11 +411,19 @@ describe('generateChangesetFromGit', () => {
           body: '',
           pr: { remote: { number: '456', author: { login: 'bob' } } },
         },
+        {
+          hash: 'cdef1234567890ad',
+          title: 'Refactored the crankshaft again',
+          body: '',
+          pr: { remote: { number: '458', author: { login: 'bob' } } },
+        },
       ],
       {},
       [
         '### Various fixes & improvements',
         '',
+        '(Only listing 3 out of 4.)',
```

Comment:
Maybe one of these alternatives with no parens and less words?

Listing 3 out of 4
_Listing 3 out of 4_
Listing **3** out of 4
Listing 3 of 4
_Showing 3 of 4_
Showing 3 of 4

Reply:
Sure thing, done in b303490feab8a05f6af717329ae20f958c26c0e2. :)

Reply:
Maybe 2ec7c16299b89e192aae735e61592edea254fe13 is even better, @billyvg?